partiql-eval/src/main/kotlin/org/partiql/eval/internal/Compiler.kt

```diff
@@ -123,6 +129,33 @@ internal class Compiler(
         return ExprPathIndex(root, index)
     }
 
+    @OptIn(PartiQLFunctionExperimental::class)
+    override fun visitRexOpCallStatic(node: Rex.Op.Call.Static, ctx: Unit): Operator {
+        val function = getFunction(node.fn.signature)
+        val args = node.args.map { visitRex(it, ctx) }.toTypedArray()
+        return ExprCallStatic(function, args)
+    }
+
+    @OptIn(PartiQLFunctionExperimental::class, PartiQLValueExperimental::class)
+    override fun visitRexOpCallDynamic(node: Rex.Op.Call.Dynamic, ctx: Unit): Operator {
+        val args = node.args.map { visitRex(it, ctx) }.toTypedArray()
+        val candidates = node.candidates.map { candidate ->
+            val fn = getFunction(candidate.fn.signature)
+            val coercions = candidate.coercions.map { it?.signature?.let { sig -> getFunction(sig) } }
+            ExprCallDynamic.Candidate(candidate.parameters.toTypedArray(), fn, coercions)
```

Comment:
Mental notes: Parameters maintains the original arg type. So 1 + 1.0 => Plus__DECIMAL__DECIMAL__DECIMAL with Parameter INT and DECIMAL.